jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/internal/jsonb/JsonbDiagnosticsParticipant.java

```diff
@@ -171,8 +171,7 @@ private boolean createJsonbTransientDiagnostic(JavaDiagnosticsContext context, S
         } else if (errorCode.equals(ErrorCode.InvalidJSonBindindAnnotationWithJsonbTransientOnAccessor)) {
             diagnosticErrorMessage = Messages.getMessage("ErrorMessageJsonbTransientOnAccessor");
         }
-        // convert to simple name for current tests
-        List<String> diagnosticData = jsonbAnnotations.stream().map(annotation -> DiagnosticUtils.getSimpleName(annotation)).collect(Collectors.toList());
+        List<String> diagnosticData = jsonbAnnotations.stream().collect(Collectors.toList());
```

Comment:
I don't understand this line. It converts a List<String> to a List<String>. Formerly it converted the names. Is this line still here because we need to make a copy of the list to pass to Gson? Or can it be removed? N.B. this construction appears a few times.

Reply:
👍